pkg/eve/Dockerfile.in

```diff
@@ -17,9 +17,15 @@ COPY --from=raw / /
 COPY --from=ipxe / /
 COPY --from=tools /out/ /
 COPY installer /bits
-COPY OVMF* /firmware/
 COPY runme.sh /
 RUN mkdir /in /out
 
+
+# These labels indicate where each component type is.
+# These must be updated if we change filenames or locations.
+# The annotations to be used are available at https://github.com/lf-edge/edge-containers/blob/master/docs/annotations.md
+LABEL org.lfedge.eci.artifact.root="/bits/rootfs.img"
+LABEL org.lfedge.eci.artifact.initrd="/bits/initrd.img"
+
 WORKDIR /bits
-ENTRYPOINT ["/runme.sh"]
+ENTRYPOINT ["//runme.sh"]
```

Comment:
why '//' ? if this a spurious change -- lets undo it

Reply:
I have no clue how that even could have gotten into there. 😆 

It must have been a typing slip, even though I don't think I ever touched that part.

Reply:
Fixed